mbuild/compound.py

```diff
@@ -984,7 +984,7 @@ def center(self):
             The cartesian center of the Compound based on its Particles
 
         """
-        if self.xyz.any():
+        if len(self.xyz) != 0:
```

Comment:
This isn't really the same logic. What was the reason for this change?

Reply:
I ran into an issue if you have one Particle located only at (0, 0, 0), that evaluates to False.

A bit more information here: ce15034

Not entirely sure the best way to handle it however.

Reply:
Ah then maybe `all(np.isfinite(self.xyz))` should do the trick. If the size of `self.xyz` isn't 3 at this point something else has gone very wrong and we should handle that issue at there.